pkg/repository/git.go

```diff
@@ -17,16 +17,21 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"path/filepath"
+	"strings"
 	"time"
 
+	"github.com/okteto/okteto/pkg/filesystem"
 	oktetoLog "github.com/okteto/okteto/pkg/log"
 
 	"github.com/go-git/go-git/v5"
 	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/object"
 )
 
 var (
 	errNotCleanRepo = errors.New("repository is not clean")
+	errNotCleanDir  = errors.New("directory is not clean")
```

Comment:
This var is unused, check with `golangci-lint run --new-from-rev d13d0c12d7fcf162c7e9dcc2c13ac3c975bc18b7`